### PR TITLE
Initial version of APB for HeptioRouter + CloudEvent bridge/converter.

### PR DESCRIPTION
**Pull request by sjwoodman**
Will deploy the following:

* openshift-event-source: HeptioRouter for obtaining the k8s events and send via HTTP
* openshift-cloud-event-source: Translate the HTTP k8s events into CE and send to CE GW ingest topic

Works but needs more development / tidying up:

* Requires EnvVarOperator - add dependency
* De-provision needs implementing

### CODE REVIEW

**Comment by sjwoodman** [comment]
Requires https://github.com/project-streamzi/openshift-cloud-event-source/pull/3

File: roles/provision-openshift-cloud-event-source-apb/tasks/provision-heptio-router.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- name: Login As Super User
+  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
```
Comment (matzew) [inline roles/provision-openshift-cloud-event-source-apb/tasks/provision-heptio-router.yml]: is that really needed  ?

Comment (sjwoodman) [reply roles/provision-openshift-cloud-event-source-apb/tasks/provision-heptio-router.yml]: Yes as it adds a clusterRole to listen to the events from k8s. :-(

Comment (matzew) [reply roles/provision-openshift-cloud-event-source-apb/tasks/provision-heptio-router.yml]: ah, that makes sense - thanks!

**Pull request merged**